jenkins/onmetal_simple_upgrade.pipeline

```diff
@@ -101,6 +101,12 @@ node('onmetal-provisioner') {
     
     //Start During Upgrade Test
     osa.start_during_test()
+
+    // Setup API uptime tests
+    osa.install_api_uptime_tests(onmetal_ip)
```

Comment:
Steve, for consistency with other function Cas asked us to remove the "onmetal_ip" parameter and to retrieve it on the groovy functions via "String host_ip = get_onmetal_ip()"
Would you mind to change it here, thanks?